src/pybamm/input/parameters/sodium_ion/Chayambuka2022.py

```diff
@@ -0,0 +1,341 @@
+    """
+    Conductivity of NaPF6 in EC:PC (1:1) as a function of ion concentration. The data
+    comes from [1].
+
+    References
+    ----------
+    .. [1] K. Chayambuka, G. Mulder, D.L. Danilov, P.H.L. Notten, Physics-based
+    modeling of sodium-ion batteries part II. Model and validation, Electrochimica
+    Acta 404 (2022) 139764. https://doi.org/10.1016/j.electacta.2021.139764.
+
+    Parameters
+    ----------
+    c_e: :class:`pybamm.Symbol`
+        Dimensional electrolyte concentration
+    T: :class:`pybamm.Symbol`
+        Dimensional temperature
+
+    Returns
+    -------
+    :class:`pybamm.Symbol`
+        Solid diffusivity
+    """
+
+    name, (x, y) = sigma_e_data
+    sigma_e = pybamm.Interpolant(x, y, c_e, name)
+
+    # Chayambuka et al. (2022) does not provide temperature dependence
+
+    return sigma_e
+
+
+# Call dict via a function to avoid errors when editing in place
+def get_parameter_values():
+    """
+    Parameters for a sodium-ion cell, from the paper :footcite:t:`Chayambuka2022` and references
+    therein. The specific parameter values are taken from the COMSOL implementation presented in
+    [this example](https://www.comsol.com/model/1d-isothermal-sodium-ion-battery-117341).
+
+    """
+
+    return {
+        "chemistry": "sodium_ion",
+        # cell
+        "Negative electrode thickness [m]": 64e-6,
+        "Separator thickness [m]": 25e-6,
+        "Positive electrode thickness [m]": 68e-6,
+        "Electrode height [m]": 2.54e-4,
+        "Electrode width [m]": 1,
+        "Nominal cell capacity [A.h]": 3e-3,
+        "Current function [A]": 3e-3,
+        "Contact resistance [Ohm]": 0,
+        # negative electrode
+        "Negative electrode conductivity [S.m-1]": 256,
+        "Maximum concentration in negative electrode [mol.m-3]": 14540,
+        "Negative particle diffusivity [m2.s-1]": HC_diffusivity_Chayambuka2022,
+        "Negative electrode OCP [V]": HC_ocp_Chayambuka2022,
+        "Negative electrode porosity": 0.51,
+        "Negative electrode active material volume fraction": 1 - 0.51 - 0.001,
+        "Negative particle radius [m]": 3.48e-6,
+        "Negative electrode Bruggeman coefficient (electrolyte)": 1.5,
+        "Negative electrode Bruggeman coefficient (electrode)": 0,
+        "Negative electrode charge transfer coefficient": 0.5,
+        "Negative electrode exchange-current density [A.m-2]"
+        "": HC_electrolyte_exchange_current_density_Chayambuka2022,
+        "Negative electrode OCP entropic change [V.K-1]": 0,
+        # positive electrode
+        "Positive electrode conductivity [S.m-1]": 50,
+        "Maximum concentration in positive electrode [mol.m-3]": 15320,
+        "Positive particle diffusivity [m2.s-1]": NVPF_diffusivity_Chayambuka2022,
+        "Positive electrode OCP [V]": NVPF_ocp_Chayambuka2022,
+        "Positive electrode porosity": 0.23,
+        "Positive electrode active material volume fraction": 1 - 0.23 - 0.22,
```

Comment:
Any reason why this is not just a single number?

Reply:
This number is not directly reported in the paper, they provide the other two and say everything adds up to one. Given that one needs to go through the paper to find this, I thought it would be better to leave it this way so it is apparent what's going on. I will add a comment to make it clear.

Reply:
Personally, I think this just looks confusing. I don't see any harm in making it a single number